woocommerce/woocommerce-admin
# Rename Homepage to Homescreen

Halfway through creating the Homepage I started to call it Homescreen because thats what the designs and internal language surrounding the project were using. 

This cleans up usage of Homepage so that there is consistency in the code base.

### Detailed test instructions:

1. Make sure the homescreen still functions.
2. Search for all instances of "homepage" in the codebase and see that they are all unrelated.

### Changelog Note:

none: changes to unreleased feature.
